user-guide/Advanced_Functionality/DataMiner_Systems/BPA_tests/BPA_SRM_Function_Resources_Consistency.md

```diff
@@ -4,20 +4,17 @@ uid: BPA_SRM_Function_Resources_Consistency
 
 # SRM Function Resources Consistency
 
-<!-- RN 35662 -->
-
-This BPA test checks for any inconsistencies between the resource objects and the [Generic DVE table] entries on the main element as well as duplicate names:
+This BPA test checks for any inconsistencies between the resource objects and the [Generic DVE table] entries on the main element as well as other possible configuration issues:
 
 - It checks for any inconsistencies in the DVE tables: states, names, linked resources, and elements.
-- It checks for duplicate names or other possible configuration issues.
+- It checks for duplicate names or IDs and other possible configuration issues.
 
-It shows the errors and warnings per element, listing the errors and warnings separately.
+It shows the global errors and errors and warnings per element, listing the errors and warnings separately.
 
-This BPA test will be included in the SRM Framework. You can [run it in System Center](xref:Running_BPA_tests) on the *Agents > BPA* tab.
+This BPA test is included in the SRM Framework. You can [run it in System Center](xref:Running_BPA_tests) on the *Agents > BPA* tab.
```

Comment:
From which version onwards is this included? It would be better to mention this.